lib/distributed_resque_worker.rb

```diff
@@ -13,9 +13,20 @@ module DistributedResqueWorker
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Layout/LineLength
 
+
+  module ResqueFailure
```

Comment:
@TruptiHosmani do we want this in this gem? Shouldnt this be in the user of the gem instead?
Also do we want to introduce bugsnag dependency in this gem?

Reply:
removed the rubocop lines and also bungsnag for this one.